optuna_dashboard/ts/components/GraphTimeline.tsx

```diff
@@ -93,11 +115,20 @@ const plotTimeline = (trials: Trial[], mode: string) => {
   }
 
   const makeTrace = (bars: Trial[], state: string, color: string) => {
+    const isRunning = state === runningKey
     const starts = bars.map((b) => b.datetime_start ?? new Date())
-    const completes = bars.map((b, i) => b.datetime_complete ?? starts[i])
+    const runDurations = bars.map((b) => {
+      const start = b.datetime_start?.getTime() ?? new Date().getTime()
+      const complete = b.datetime_complete?.getTime() ?? start
```

Comment:
How about this since we can use the current time for running trials?
```
const complete = b.datetime_complete?.getTime() ?? maxDatetime.getTime()
```

Reply:
`start` is necessary for waiting trials.